crates/starknet-types-core/src/felt/mod.rs

```diff
@@ -1062,6 +1062,18 @@ mod errors {
     }
 }
 
+mod zeroing {
```

Comment:
you don't really need to create a new module here. Or if you do, it should be behind the feature flag. Here it will happen in the path, even if it is empty, when the feature is not enabled

Reply:
I would recommend to make it a file. `zeroize.rs`. And put `mod` import behind the feature flag in felt/mod.rs

Reply:
felt/mod.rs is a beefy 2k lines of code. It's already painful to navigate it

Reply:
But it's just a matter of taste of preference. So I won't force mine on the codebase